packages/components/src/radio/docs/radiogroup/orientation.tsx

```diff
@@ -0,0 +1,20 @@
+import { Inline, RadioGroup, Radio, RadioList } from "@hopper-ui/components";
+
+export default function Example() {
+    return (
+        <Inline>
```

Comment:
Can we add a bigger gap here? There's just something that's been bugging me about the display and I think it might be because the gap between the two lists is about the same size as the gap between Radios.

Reply:
Ya, I tried to avoid unnecessary boilerplate code for the user, to focus on the code needed but it was bugging me too